redpanda/ci/01-one-node-cluster-values.yaml

```diff
@@ -14,9 +14,3 @@
 # limitations under the License.
 statefulset:
   replicas: 1
-  resources:
-    limits:
-      memory: 1Gi
-  redpanda:
-    memory: 700M
-    reservedMemory: 200M
```

Comment:
These values are removed since there are checks on memory given to Redpanda and the container; if theses are lower than recommended values (2Gi per core minimum), then an error is thrown: https://github.com/vuldin/helm-charts/blob/5bcb118e5dacac8aa154a351ae8c6851187931aa/redpanda/templates/_helpers.tpl#L381-L386